pixi.toml

```diff
@@ -77,14 +77,14 @@ lint-rs-all = "cargo fmt --check"
 lint-py-fmt-check = "ruff format --check --config rerun_py/pyproject.toml"
 lint-py-blackdoc = "blackdoc --check"
 lint-py-mypy = "mypy --install-types --non-interactive --no-warn-unused-ignore"
-lint-py-ruff = "ruff check --config rerun_py/pyproject.toml"
+lint-py-ruff = "ruff format --check --config rerun_py/pyproject.toml"
 lint-taplo = "taplo fmt --check --diff"
 lint-typos = "typos"
 
 misc-fmt = "prettier --write '**/*.{yml,yaml,js,css,html}'"
 misc-fmt-check = "prettier --check '**/*.{yml,yaml,js,css,html}'"
 toml-fmt = "taplo fmt"
-ruff-fix = "ruff --fix --config rerun_py/pyproject.toml ."
+ruff-fix = "ruff format --config rerun_py/pyproject.toml ."
```

Comment:
is `ruff format` and `ruff --fix` really the same thing? Was it renamed?

Reply:
I'm confused by this as well - does `ruff --fix` work for you? Because on Windows this does nothing for me, so it's clearly not the same but it didn't get so much renamed as to the command failing -.-

Reply:
I believe `ruff --fix` fixes lints, i.e. NOT formatting.

So I would suggest keeping `ruff-fix` and `adding `ruff-format`, or combine them both under one command

Reply:
ah! adding both; can add a meta command later